scispacy/abbreviation.py

```diff
@@ -145,14 +147,12 @@ class AbbreviationDetector:
     Note that this class does not replace the spans, or merge them.
     """
 
-    def __init__(self, nlp) -> None:
+    def __init__(self, nlp: Language, name: str = "") -> None:
```

Comment:
Why the blank string rather than `abbreviation_detector`?

Reply:
it can be whatever we want. the arg only gets used when spacy internals calls this `__init__` function, and it is set via the `Language.factory` at the top of the class. i only gave it a default here, because sometimes we initialize this class ourselves, and it is kind of weird to have to pass in a name that we never use. but the default can be anything.